astroquery/simbad/core.py

```diff
@@ -936,6 +1006,243 @@ def query_objectids_async(self, object_name, *, cache=True,
 
         return response
 
+    def tables(self, get_adql=False):
+        """The names and descriptions of the tables in SIMBAD.
+
+        Parameters
+        ----------
+        get_adql : bool, optional
+            Returns the ADQL string instead of querying SIMBAD.
+
+        Returns
+        -------
+        `~astropy.table.table.Table`
+        """
+        query = ("SELECT table_name, description"
+                 " FROM TAP_SCHEMA.tables"
+                 " WHERE schema_name = 'public'")
+        if get_adql:
+            return query
+        return self.query_tap(query)
+
+    def columns(self, *tables: str, get_adql=False):
```

Comment:
Same, add a verb for the method name to better describe it, `list_columns` maybe?